docs/install-run/special/linux.txt

```diff
@@ -4,15 +4,22 @@
 Install CrateDB on Linux
 ========================
 
-CrateDB maintains official packages for Debian, RedHat Enterprise Linux, and
-Ubuntu.
+CrateDB maintains packages for `Debian GNU/Linux`_, `Ubuntu`_, and `RedHat
+Linux`_.
 
-We recommend that you use our one-step installer that will configure the
-appropriate package repository and install CrateDB for you.
+.. _Debian GNU/Linux: https://crate.io/docs/crate/guide/en/latest/deployment/linux/debian.html
+.. _Ubuntu: https://crate.io/docs/crate/guide/en/latest/deployment/linux/ubuntu.html
+.. _RedHat Linux: https://crate.io/docs/crate/guide/en/latest/deployment/linux/red-hat.html
+
+You can take `the manual approach`_, but we recommend that you use our one-step
+installer that will configure the appropriate package repository and install
+CrateDB for you.
```

Comment:
this installer is only recommended for "try out" purposes, not for production deployments.

Reply:
@chaudum made some bigger changes based on your feedback. can you review again? please and thank you!